ngr_spider/util.py

```diff
@@ -388,12 +391,24 @@ def get_wmts_service(
     service_record: CswServiceRecord,
 ) -> Union[WmtsService, ServiceError]:
     def convert_layer(lyr) -> WmtsLayer:
+        styles: list[Style] = []
```

Comment:
Ik vermoed dat het converteren van de style nu in een gedeelde functie kan voor zowel WMS als WMTS. Maar is op zich geen must.